problem with 1.11.0

Arch GNU/Linux

```
	mkdir build
	cd build
	cmake ..
	make rhubarb
```

```
CMake Warning (dev) in CMakeLists.txt:
  No project() command is present.  The top-level CMakeLists.txt file must
  contain a literal, direct call to the project() command.  Add a line of
  code such as

    project(ProjectName)

  near the top of the file, but after cmake_minimum_required().

  CMake is pretending there is a "project(Project)" command on the first
  line.
This warning is for project developers.  Use -Wno-dev to suppress it.

[...]

hubarb-lip-sync-master/rhubarb/src/tools/exceptions.cpp: En la función ‘std::string getMessage(const std::exception&)’:
rhubarb-lip-sync-master/rhubarb/src/tools/exceptions.cpp:9:17: error: ‘rethrow_if_nested’ no se declaró en este ámbito
    9 |                 rethrow_if_nested(e);

```

CMake Warning (dev) in CMakeLists.txt:
  No project() command is present.  The top-level CMakeLists.txt file must
  contain a literal, direct call to the project() command.  Add a line of
  code such as

    project(ProjectName)

  near the top of the file, but after cmake_minimum_required().

  CMake is pretending there is a "project(Project)" command on the first
  line.
This warning is for project developers.  Use -Wno-dev to suppress it.

/home/bazza/src/aur/rhubarb-lip-sync/src/rhubarb-lip-sync-1.11.0/rhubarb/src/tools/exceptions.cpp: In function 'std::string getMessage(const std::exception&)':
/home/bazza/src/aur/rhubarb-lip-sync/src/rhubarb-lip-sync-1.11.0/rhubarb/src/tools/exceptions.cpp:9:17: error: 'rethrow_if_nested' was not declared in this scope
    9 |                 rethrow_if_nested(e);
      |                 ^~~~~~~~~~~~~~~~~
make[3]: *** [rhubarb/CMakeFiles/rhubarb-tools.dir/build.make:76: rhubarb/CMakeFiles/rhubarb-tools.dir/src/tools/exceptions.cpp.o] Error 1
make[3]: *** Waiting for unfinished jobs....
/home/bazza/src/aur/rhubarb-lip-sync/src/rhubarb-lip-sync-1.11.0/rhubarb/src/tools/ProgressBar.cpp: In member function 'void ProgressBar::updateLoop()':
/home/bazza/src/aur/rhubarb-lip-sync/src/rhubarb-lip-sync-1.11.0/rhubarb/src/tools/ProgressBar.cpp:43:35: error: 'sleep_for' is not a member of 'std::this_thread'
   43 |                 std::this_thread::sleep_for(animationInterval);
      |                                   ^~~~~~~~~
make[3]: *** [rhubarb/CMakeFiles/rhubarb-tools.dir/build.make:146: rhubarb/CMakeFiles/rhubarb-tools.dir/src/tools/ProgressBar.cpp.o] Error 1
make[2]: *** [CMakeFiles/Makefile2:366: rhubarb/CMakeFiles/rhubarb-tools.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:525: rhubarb/CMakeFiles/rhubarb.dir/rule] Error 2
make: *** [Makefile:293: rhubarb] Error 2
==> ERROR: A failure occurred in prepare().
    Aborting...

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2,6 +2,8 @@ cmake_minimum_required(VERSION 3.2)
 
 include(appInfo.cmake)
 
+project(${appName})
+
 # Build and install main executable
 add_subdirectory(rhubarb)
 

--- a/rhubarb/CMakeLists.txt
+++ b/rhubarb/CMakeLists.txt
@@ -5,8 +5,6 @@ include("../appInfo.cmake")
 # Support legacy OS X versions
 set(CMAKE_OSX_DEPLOYMENT_TARGET "10.10" CACHE STRING "Minimum OS X deployment version")
 
-project(${appName})
-
 # Enable C++17
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)

--- a/rhubarb/src/tools/ProgressBar.cpp
+++ b/rhubarb/src/tools/ProgressBar.cpp
@@ -1,10 +1,12 @@
 #include "ProgressBar.h"
+
 #include <algorithm>
 #include <future>
 #include <chrono>
 #include <format.h>
 #include <boost/algorithm/clamp.hpp>
 #include <cmath>
+#include <thread>
 
 using std::string;
 

--- a/rhubarb/src/tools/exceptions.cpp
+++ b/rhubarb/src/tools/exceptions.cpp
@@ -1,3 +1,5 @@
+#include <exception>
+
 #include "exceptions.h"
 
 using std::string;

